src/aria/widgets/form/CheckBox.js

```diff
@@ -229,10 +243,12 @@ module.exports = Aria.classDefinition({
         _onBoundPropertyChange : function (propertyName, newValue, oldValue) {
             if (propertyName === 'value') {
                 this._cfg.value = newValue;
+                this._updateWAIAttributes('aria-checked', newValue);
```

Comment:
This line can be removed, as it is also done in `_updateFomForState()` which is called line 248.